proto/quasar/tokenfactory/v1beta1/query.proto

```diff
@@ -0,0 +1,124 @@
+syntax = "proto3";
+package quasar.tokenfactory.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "quasar/tokenfactory/v1beta1/authorityMetadata.proto";
+import "quasar/tokenfactory/v1beta1/params.proto";
+
+option go_package = "github.com/quasar-finance/quasar/x/tokenfactory/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // Params defines a gRPC query method that returns the tokenfactory module's
+  // parameters.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http) = {
+      get: "/quasar/tokenfactory/v1beta1/params"
+      additional_bindings {
```

Comment:
Why is this additional binding required with the previous values proto paths? Is there any constrained do we have.

Reply:
@ajansari95

Reply:
so we are backward compatible with old routes, if somebody has workflow it doesn't break. We can get rid of it later.Or if we dont care about backward compatibility we can remove.